docs/CONFIG-VARS.md

```diff
@@ -191,22 +190,49 @@ When setting V4_CFG_TLS_MODE to a value other than "disabled" and no V4_CFG_TLS_
-| V4_CFG_POSTGRES_TYPE | Postgres installation type | string | | true | [internal,external] | viya |
+Postgres servers can be defined with the postgres_servers variable which is a map of objects. The variable has the following format:
+
+```bash
+V4_CFG_POSTGRES_SERVERS:
+  default: {}
+  ...
+```
 
-### External Postgres
+**NOTE**: the `default` elements is always required . This will be the default server. Below is the list of parameters each element can contain.
 
 | Name | Description | Type | Default | Required | Notes | Tasks |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
-| V4_CFG_POSTGRES_ADMIN_LOGIN | Existing postgres username | string | | true | | viya |
-| V4_CFG_POSTGRES_PASSWORD | Existing postgres password | string | | true | | viya |
-| V4_CFG_POSTGRES_FQDN | Existing postgres ip/fqdn | string | | true | | viya |
-| V4_CFG_POSTGRES_PORT | Existing postgres port | string | 5432 | false | | viya |
-| V4_CFG_POSTGRES_DATABASE | Existing postgres database name | string | "SharedServices" | false | | viya |
-| V4_CFG_POSTGRES_SSL_ENFORCEMENT | Require ssl connection to existing postgres | bool | false | false | Ignored on GCP when using cloud sql | viya |
-| V4_CFG_POSTGRES_CONNECTION_NAME | Existing postgres database connection name | string | | false | See [ansible cloud authentication](user/AnsibleCloudAuthentication.md) | viya |
-| V4_CFG_POSTGRES_SERVICE_ACCOUNT | Existing service account for postgres connectivity | string | | false | See [ansible cloud authentication](user/AnsibleCloudAuthentication.md) | viya |
+| internal | Whether the database is internal or external | bool | | true | All servers must but internal or all must be external | viya |
+| database | Database name | string | Database server role | false | Default database name for default server is SharedServices | viya |
+| admin | External postgres username | string | | false | Required for external postgres servers | viya |
+| password | External postgres password | string | | false | Required for external postgres servers | viya |
+| fqdn | External postgres ip/fqdn | string | | false | Required for external postgres servers | viya |
+| server_port | External postgres port | string | 5432 | false | | viya |
+| ssl_enforcement_enabled | Require ssl connection to external postgres | bool | | false | Required for external postgres servers. Ignored on GCP when using cloud sql | viya |
+| connection_name | External postgres database connection name | string | | false | Required for using cloud-sql-proxy on gcp. See [ansible cloud authentication](user/AnsibleCloudAuthentication.md) | viya |
+| service_account | External service account for postgres connectivity | string | | false | Required for using cloud-sql-proxy on gcp. See [ansible cloud authentication](user/AnsibleCloudAuthentication.md) | viya |
+
+Example:
+
+```bash
+V4_CFG_POSTGRES_SERVERS:
+  default:
+    internal: false
+    admin: pgadmin
+    password: password
```

Comment:
password: "password"

Add quotes to the example as this is how we want customers to pass it in.